src/services/interfaceGenerator/InterfaceGenerator.ml

```diff
@@ -0,0 +1,240 @@
+let spf = Printf.sprintf
```

Comment:
- missing copyright header
- filename should be interface_generator.ml and the module should be `Interface_generator`